chart/values.yaml

```diff
@@ -13,6 +13,8 @@ images: # images and tag to pull for each component of the stack
 controller:
   deleteOnly: false # enable delete-only mode
   metricsSink: noop # metrics driver (noop or datadog)
+  defaultDuration: 1h # default spec.duration for a disruption with none specified
```

Comment:
Good to keep in mind that changing this delay will NOT affect existing disruptions (and it is the expected behavior anyway).